music21/braille/translate.py

```diff
@@ -55,6 +55,10 @@
     that should be present in a line of braille.
 * **segmentBreaks** ([]): A list consisting of (measure number, offset start)
     tuples indicating where the music should be broken into segments.
```

Comment:
That wouldn't be difficult to implement -- I think we're talking about a for-loop and then
`obj.measure(x).insert(y, BrailleSegmentDivision())` so pretty trivial.

Given that, I'm inclined to just comment the keywords back in and let a subsequent PR implement that feature and any tests it needs, etc